.github/workflows/docker-hub-latest.yml

```diff
@@ -26,10 +31,12 @@ jobs:
 
       # Needed for multi platform builds
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3.0.0
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: ${{ env.PLATFORMS }}
```

Comment:
where are we defining this?

Reply:
further up in the file(s) under the `env` key